pkg/profiler/cpu/maps.go

```diff
@@ -1118,10 +1126,28 @@ func (m *bpfMaps) cleanStacks() error {
 	return result
 }
 
+func (m *bpfMaps) cleanEventsCount() error {
+	if err := clearBPFMap(m.eventsCount); err != nil {
+		return err
```

Comment:
Since https://github.com/parca-dev/parca-agent/pull/2041 is merged, could we add a metric for the clean as well?

Reply:
Great shout!